.github/workflows/linter.yaml

```diff
@@ -0,0 +1,24 @@
+name: Pylint
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pylint
+        pip install pyGitHub
+    - name: Analysing the code with pylint
+      run: |
+        find . -type f -name "*.py" | xargs pylint --exit-zero
+        
```

Comment:
add a newline.

Reply:
A second new line to the end of the file, or where do you mean?

Reply:
github is indicating that the file did not end with a newline.

Reply:
@klfgit test

Reply:
![Screen Shot 2021-11-18 at 8 40 49 AM](https://user-images.githubusercontent.com/2697864/142426196-aa2eb9a7-180f-41fe-be87-833bacebc667.png)

@jacobeicher  ^^

Reply:
![image](https://user-images.githubusercontent.com/46536223/142689937-1e2a1dcb-0a4a-40a9-a83a-054b8e60de7b.png)

That's very strange, it looks like its there in the request and Its there on the branch